### PR TITLE
 EOS-26885-Motr user mode only rpms are available

### PR DESCRIPTION
**Pull request by nikhilpatil2995**
# Problem Statement
-  EOS-26885-Motr user mode only rpms are available

# Design
-  For Bug, Build motr rpm without using kernel and lustre package.
-  For Feature, Post the link for design

# Coding
   Checklist for Author
-  [ ] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
- [ ] Unit and System Tests are added
- [ ] Test Cases cover Happy Path, Non-Happy Path and Scalability. Tested with - http://eos-jenkins.colo.seagate.com/job/Release_Engineering/job/re-workspace/job/custom-ci-test-EOS-25657/104/console
- [ ] Testing was performed with RPM

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
- [ ] Interface change (if any) are documented
- [ ] Side effects on other features (deployment/upgrade)
- [ ] Dependencies on other component(s)

# Review Checklist 
  Checklist for Author
- [ ] JIRA number/GitHub Issue added to PR
- [ ] PR is self reviewed
- [ ] Jira and state/status is updated and JIRA is updated with PR link
- [ ] Check if the description is clear and explained

# Documentation
  Checklist for Author
- [ ] Changes done to WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

File: jenkins/internal-ci/centos-7.9.2009/branch/motr.groovy [hunk 1]
```diff
@@ -87,7 +87,7 @@ pipeline {
                         rm -rf /root/rpmbuild/RPMS/x86_64/*.rpm
                         KERNEL=/lib/modules/$(yum list installed kernel | tail -n1 | awk '{ print $2 }').x86_64/build
```
Comment (shailesh-vaidya) [inline jenkins/internal-ci/centos-7.9.2009/branch/motr.groovy:88]: Please remove this line also.

File: jenkins/custom-ci/centos-7.9.2009/motr.groovy [hunk 2]
```diff
@@ -67,7 +67,7 @@ pipeline {
 						rm -rf /root/rpmbuild/RPMS/x86_64/*.rpm
 						KERNEL=/lib/modules/$(yum list installed kernel | tail -n1 | awk '{ print $2 }').x86_64/build
```
Comment (shailesh-vaidya) [inline jenkins/custom-ci/centos-7.9.2009/motr.groovy:68]: Please remove this line also.

**Review by shailesh-vaidya — CHANGES_REQUESTED** [review]

File: jenkins/custom-ci/centos-7.9.2009/motr.groovy (hunk 2 above)
Comment (nikhilpatil2995) [reply jenkins/custom-ci/centos-7.9.2009/motr.groovy:68]: but In actual file I can't see this line  https://github.com/nikhilpatil2995/cortx-re-1/blob/EOS-26885-motr-user-mode-rpms/jenkins/custom-ci/centos-7.9.2009/motr.groovy

Comment (nikhilpatil2995) [reply jenkins/custom-ci/centos-7.9.2009/motr.groovy:68]: okay

**Review by shailesh-vaidya — COMMENTED** [review]
@nikhilpatil2995  - Jenkins pipeline shows below error - 
```
10:36:29  + cp cortx-motr.spec.in cortx-motr.spec
10:36:29  + sed -i 's/@.*@/111/g' cortx-motr.spec
10:36:29  + yum-builddep -y cortx-motr.spec
10:36:29  Loaded plugins: fastestmirror, ovl, priorities
10:36:39  Determining fastest mirrors
10:36:39  error: line 132: Unknown tag: 111
10:36:39  Bad spec: cortx-motr.spec
```

This will cause issue if new package is added in spec file. Please check.

**Comment by nikhilpatil2995** [comment]
> @nikhilpatil2995 - Jenkins pipeline shows below error -
> 
> ```
> 10:36:29  + cp cortx-motr.spec.in cortx-motr.spec
> 10:36:29  + sed -i 's/@.*@/111/g' cortx-motr.spec
> 10:36:29  + yum-builddep -y cortx-motr.spec
> 10:36:29  Loaded plugins: fastestmirror, ovl, priorities
> 10:36:39  Determining fastest mirrors
> 10:36:39  error: line 132: Unknown tag: 111
> 10:36:39  Bad spec: cortx-motr.spec
> ```
> 
> This will cause issue if new package is added in spec file. Please check.

@shailesh-vaidya, This error is not due to current changes.

**Review by shailesh-vaidya — DISMISSED** [review]

**Review by gauravchaudhari02 — APPROVED** [review]

**Review by shailesh-vaidya — APPROVED** [review]

**Pull request merged**